src/components/CustomModal/style.css

```diff
@@ -2,61 +2,87 @@
 .modal :global(.ant-modal-content) {
     /* prevents modal body color rendering under header/footer at corners */
     border-radius: 3px;
+    color: var(--light-theme-modal-color);
+}
+
+.modal :global(.ant-modal-header) {
+    display: flex;
+    height: 46px;
+    justify-content: center;
 }
 
 .modal :global(.ant-modal-title) {
-    text-align: center;
-    font-weight: bold;
-    font-size: 14px;
+    align-items: center;
+    display: flex;
+    font-size: 16px;
+    font-weight: 400;
 }
 
-.modal :global(.ant-modal-footer) {
-    padding: 12px 16px 20px;
-    background-color: var(--light-theme-modal-content-bg);
-    max-height: 56px;
+.modal :global(.close-icon) {
+    cursor: pointer;
+    display: flex;
+    position: absolute;
+    right: 24px;
+}
+
+.modal :global(.close-icon):hover::after {
+    color: var(--light-theme-modal-close-icon-hover);
 }
 
 .modal :global(.ant-modal-body) {
-    padding: 24px 24px 0px 24px;
     font-size: 16px;
-    color: var(--dark);
+    padding: 24px 24px 10px 24px;
 }
 
-.modal :global(.ant-modal-header) {
-    height: 40px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+.modal.divider :global(.ant-modal-body)  {
+    padding: 24px;
```

Comment:
these two padding rules to ensure proper spacing for divider and non/divider modals